### PR TITLE
Add test cases "Change assigner" and move wait time in base class

### PR DESCRIPTION
**Pull request by vitaliisotnichenko**

### CODE REVIEW

File: src/pages/create_issue_page.py [hunk 1]
```diff
@@ -18,44 +17,41 @@ def should_have_title(self):
                 __create_issue_title = self.browser.find_element(By.CSS_SELECTOR, "[title=\"Create Issue\"]").text
```
Comment (archick12) [inline src/pages/create_issue_page.py]: Please, replcae "\ with ' symbol

File: src/pages/issue_details_page.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import NoSuchElementException, StaleElementReferenceException, \
+    ElementNotInteractableException, ElementClickInterceptedException
+
+from src.pages.base_page import BasePage
+
+class IssueDetailsPage(BasePage):
+
+    def is_edit_button_at_page(self):
```
Comment (archick12) [inline src/pages/issue_details_page.py]: Please, replace is_edit_button_at_page to is_edit_button_present()

File: src/pages/issue_details_page.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                    self.browser.find_element(By.CSS_SELECTOR, "#assignee-field").send_keys(name)
+                    self.browser.find_element(By.CSS_SELECTOR, "#assignee-field").send_keys(Keys.ENTER)
+                    self.browser.find_element(By.CSS_SELECTOR, ".aui-iconfont-success").click()
+
+                except (NoSuchElementException, StaleElementReferenceException, ElementNotInteractableException,
+                    ElementClickInterceptedException):
+                        time.sleep(5)
+                        i += 1
+
+    def should_be_new_assigner(self):
```
Comment (archick12) [inline src/pages/issue_details_page.py]: I see here 2 options 1) should_be_new_assigner should return boolean 
2) return text, but the name should be changed to "get_assignee_name()"

I suggest 1st option - return boolean. Add new parameter expected_assignee and compare expected with actual values inside the method. Then assert the result at the test level.

File: src/pages/issue_details_page.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        for i in range(3):
+            try:
+                __is_new_assigner = self.browser.find_element(By.CSS_SELECTOR, "#assignee-val>span[class='user-hover']")
+                return __is_new_assigner.text
+            except (NoSuchElementException, StaleElementReferenceException, ElementClickInterceptedException,
+                    ElementNotInteractableException):
+                time.sleep(5)
```
Comment (archick12) [inline src/pages/issue_details_page.py]: '5' must be a class field called 'waitTime'

File: src/pages/main_page.py [hunk 5]
```diff
@@ -11,6 +15,17 @@ def is_create_issue_button(self):
...
+
+    def click_the_first_ticket_assigned_to_me(self):
+        for i in range(3):
+            try:
+                __the_first_issue = self.browser.find_element_by_css_selector(".hide-carrot tr:nth-child(1)[class='issuerow']>td+[class=issuekey]")
+                if __the_first_issue.is_displayed():
+                    return __the_first_issue.click()
+
+            except (NoSuchElementException, StaleElementReferenceException, ElementClickInterceptedException, ElementNotInteractableException):
+                time.sleep(5)
```
Comment (archick12) [inline src/pages/main_page.py]: Move  this value to a class field 'sleepTimeForRetry'
Create a dictionary with values 'slow' -> 20 seconds, 'very fast' -> 1 second,  'fast' -> 5 seconds, 'medium' -> 10

In the end your code should look like "time.sleep(sleepTimeForRetry["fast"])"

**Comment by vitaliisotnichenko** [comment]
Update code according to requirements

**Pull request merged**